rustls-libssl/src/verifier.rs

```diff
@@ -161,3 +143,126 @@ impl ServerCertVerifier for ServerVerifier {
             .supported_schemes()
     }
 }
+
+#[derive(Debug)]
+pub struct ClientVerifier {
+    parent: Arc<dyn ClientCertVerifier>,
+    mode: VerifyMode,
+    last_result: AtomicI64,
+}
+
+impl ClientVerifier {
+    pub fn new(
+        root_store: Arc<RootCertStore>,
+        provider: Arc<CryptoProvider>,
+        mode: VerifyMode,
+    ) -> Result<Self, Error> {
+        let (parent, initial_result) = if !mode.server_must_attempt_client_auth() {
+            (Ok(WebPkiClientVerifier::no_client_auth()), X509_V_OK)
+        } else {
+            let builder = WebPkiClientVerifier::builder_with_provider(root_store, provider)
+                .allow_unknown_revocation_status();
```

Comment:
Since CRLs aren't being used I don't think `.allow_unknown_revocation_status();` is particularly meaningful. Did you include it for a reason I'm overlooking?

Reply:
Misreading the docs! :) Removed.